components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/resources/applications.yaml

```diff
@@ -2800,6 +2794,12 @@ components:
           type: integer
           format: int64
           example: 3600
+        bindingType:
```

Comment:
Can we set the default value to 'None'?

We can either do this at the API swagger or at the code.

Reply:
Updated with https://github.com/wso2/identity-api-server/pull/186/commits/9a85e7ed286f2a7422abc510521aaa02ccf0fd4a